### ISSUE TITLE
Stick To setting does not persist to Shift+Select Compare or Compare Selected

### ISSUE THREAD

**Issue by fireundubh** [TES5Edit/TES5Edit#666]
@zilav 
When you compare multiple records using Shift+Select or Compare Selected, if a Stick To setting had been set previously, the setting does not persist to the comparison view.

**Comment by zilav** [comment]
Yep, originally it was added by request when in browsing records "mode".
